### DISCUSSION TITLE
Tic-80 and multi cartridge

### DISCUSSION THREAD

**Discussion by desC-37** [nesbox/TIC-80#1420] [Q&A] [answered]
Hello!
I know there is multicatridge at pico 8. Is there anything equivalent on the Tic-80?

**Comment by nesbox** [comment, marked as answer]
Hi,
No, there is no such feature, but TIC-80 has Bank Switching https://github.com/nesbox/TIC-80/wiki/sync

**Reply by desC-37** [reply]
thank you!
